Content/install/home.htm

```diff
@@ -18,6 +19,32 @@ <h1>Installing <MadCap:variable name="General.Liquibase" /></h1>
                 <MadCap:xref href="liquibase-linux.htm">Installing [%=General.Liquibase%] on  Linux/Unix</MadCap:xref>
             </li>
         </ul>
+
+        <h2>Software Configuration</h2>
+        <p>The <MadCap:variable name="General.Liquibase" /> CLI ships with drivers for many popular database systems, including:</p>
+        <ul>
+            <li>DB2</li>
+            <li>H2</li>
+            <li>HyperSQL</li>
+            <li>Firebird</li>
+            <li>MariaDB</li>
+            <li>MS SqlServer</li>
+            <li>Oracle</li>
+            <li>PostgreSQL</li>
+            <li>SQLite</li>
+        </ul>
+
+        <p>If you are using a different database system or need to run a different version of the driver, you can download the appropriate driver from your vendor,
+        and install it in one of two places:</p>
+        <ul>
+            <li>The "lib" directory of your <MadCap:variable name="General.Liquibase" /> installation -- Drivers added here will always be used by <MadCap:variable name="General.Liquibase" /></li>
+            <li>A "liquibase_libs" directory of your current working directory -- Drivers added here will only be used when you are in this working directory</li>
```

Comment:
The liquibase_libs directory is one that a user must add themselves, correct? Or is it created by the user? If it is that the user needs to create liquibase_libs, we should make that explicit. It would be also be good to explain a common use case. As it is, there's not a clear distinction as to why I'd want to use one or the other.
CC @nvoxland

Reply:
It's a directory users have to create themselves in whatever working directories they run liquibase from. 

The use case is "I have two separate liquibase projects, and one of them needs an extension or driver that causes problems with the other", so I want jars local to just this project.